src/components/Editor/ItemEditor.vue

```diff
@@ -777,12 +767,10 @@ export default {
         this.selectedItemDetail.max_char_limit = value;
       },
     },
-    isSelectedItemSurveyQuestion: {
-      get() {
-        // extract whether question is for survey
-        if (this.selectedItemDetail == null) return false;
-        return this.selectedItemDetail.survey;
-      },
+    isSelectedItemSurveyQuestion() {
+      // extract whether question is for survey
```

Comment:
Sorry for not commenting earlier! I think you can safely remove this comment as the name of the method makes it pretty obvious